test-math/Cargo.toml

```
@@ -7,6 +7,7 @@ edition = "2021"
 cc = "1.0"
 
 [dependencies]
+approx = "0.5.1"
```

```suggestion
approx = "0.5"
```

I prefer the compat bounds cleaner.

Actually can we move all of this to `dev-dependencies` since we are using for `cargo test`.

If so, adjust the compat of `approx` there as well.